tests/integration/cloud/rest_contracts/unit_test_mocks/conftest.py

```diff
@@ -0,0 +1,61 @@
+import json
+import uuid
+from unittest import mock
+
+import pytest
+import requests
+from pact.matchers import Matcher
+
+import great_expectations as gx
+from great_expectations.data_context import CloudDataContext
+from tests.integration.cloud.rest_contracts.conftest import JsonData, PactBody
+from tests.integration.cloud.rest_contracts.test_data_context_configuration import (
+    DATA_CONTEXT_CONFIGURATION_MIN_RESPONSE_BODY,
+)
+
+
+def _convert_matcher_to_value(matcher: Matcher) -> JsonData:
+    return matcher.generate()["contents"]
+
+
+def _reify_pact_body(
+    body: PactBody,
+) -> JsonData:
+    if isinstance(body, list):
+        for index, item in enumerate(body):
+            if isinstance(item, Matcher):
+                body[index] = _convert_matcher_to_value(matcher=item)
+            body[index] = _reify_pact_body(body=body[index])
+        return body
+    elif isinstance(body, Matcher):
+        return _reify_pact_body(body=_convert_matcher_to_value(matcher=body))
+    elif isinstance(body, dict):
+        for key, value in body.items():
+            if isinstance(value, Matcher):
+                body[key] = _convert_matcher_to_value(matcher=value)
+            body[key] = _reify_pact_body(body=body[key])
+        return body
+    else:
+        return body
```

Comment:
```suggestion
    return body
```

Reply:
I like the original way better. Sure this is less code, but it requires my brain to think slightly more.